CONTRIBUTING.md

```diff
@@ -232,3 +232,39 @@ releases will be based off of that release.
 - delete the `pre-rc/vX.X.X` and `RC` branches
 - create a PR into `master` containing ONLY the CHANGELOG.md updates
 - tag and release `release/vX.XX.X`
+
+## Code Owner Membership 
+
+In the ethos of open source projects, and out of necessity to keep the code
+alive, the core contributor team will strive to permit special repo privileges
+to developers who show an aptitude towards developing with this code base. 
+
+Several different kinds of privileges may be granted however most common
+privileges to be granted are merge rights to either part of, or the entire the
+code base (though the github `CODEOWNERS` file). The on-boarding process for
+new code owners is as follows: On a bi-monthly basis (or more frequently if
+agreeable) all the existing code owners will privately convene to discuss
```

Comment:
Do we have a email list that is just for committers? That could be the easiest way to start a vote and it’s archived as record

Reply:
Voting should happen during an agreeable conference phone/video meeting _not_ by email. The members can then all approve a PR which modifies the CODE_OWNERS file to publicly show their support for a change. This PR should take place after the meeting is held, and unanimous consensus reached

Reply:
Added the following text: 
```
This private meeting is to be a held as a
phone/video meeting. Subsequently at the end of the meeting, one of the existing
code owners should open a PR modifying the `CODEOWNERS` file. The other code
owners should then all approve this PR to publicly display their support.
```